app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+import os
+from models import Actor
+
+
+ABSOLUTE_PATH = os.path.abspath("../cinema.sqlite")
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(ABSOLUTE_PATH)
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
```

Comment:
It's a good practice to handle exceptions for database operations to ensure that any issues during the execution of SQL commands are caught and managed appropriately.